### ISSUE TITLE
Types of Locations

### ISSUE THREAD

**Issue by Treijim** [bheiskell/Riiga#42]
Since I'm writing the locations up, I think it's a good idea to create a set of icons for locations which are displayed next to the location name. These icons indicate things like, who governs/runs the place, what sort of place it is (large/small city, capital, forest, temple, etc.), how dangerous it is (low, medium, high, etc.), and possibly other things.

**Comment by Treijim** [comment]
Also thought of other things: some places are outposts or fortresses, and there should be a symbol used for places which are governmentally corrupt or officially abandoned. 

All such symbols will provide brief roll-over information.

**Comment by bheiskell** [comment]
We need to talk about where you want this. I'm for more information, but keeping visual clutter to a minimum should remain a high priority too. :-)

As far as technical implementation, I think we need location tags. Keywords that you as an admin can tag onto a location. These keywords would have an icon and description you write up explaining what they offer.

This is more extension. Do you want this as a wishlist feature or something for the beta roll out? (Btw, I've created a wishlist tag to use for items that wont be rolled out in the beta release.)

**Comment by bheiskell** [comment]
Oh, danger doesn't have to be a tag. That could be its own field. It would give me an excuse to use the following jQuery plugin. ;-D

http://www.filamentgroup.com/lab/update_jquery_ui_slider_from_a_select_element_now_with_aria_support/
See the slider right before the comments section.

**Comment by Treijim** [comment]
I actually see this as quite important; the icons/keywords provide important data about places and save a lot of time, questions, and confusion (hopefully). If I add keywords onto the descriptions of the locations, could you then interpret that data and display it as icons? Or would it be easier to make a list of checkboxes on the "new location" page? 

I don't see a use for that slider, but it does depend on how I input the keywords. What's a simple and effective way to add keywords to the location creation page? I think this should be done before I make any more locations, or else I just have to go back and edit more locations.

**Comment by bheiskell** [comment]
Slider would be for the low to high danger.

Keywords would have to be created separate from the locations. Each location tag would have its own icon. Then when you tag a location with that location tag, it would receive the icon. You would add the location tag on the edit/add location page.

**Comment by Treijim** [comment]
But is the slider purely for when I input the level of danger? Nobody else can see it? What's the difference between having that and having the relevant keyword or a radio button for my input?

So keywords are to be added using a keyword text input area? Separated by commas? Let me know when you add this.

**Comment by bheiskell** [comment]
There is no purpose except it's shiny. :-) I wont use it.

Probably a multiselect since that's why cakephp defaults to. You would ctrl click the keywords you wanted. Alternatively it could be check boxes. Which is easier for you?

**Comment by bheiskell** [comment]
Creating tags is now available at alpha/location_tags. Use the select widget on the add/edit location page to assign the tags to the location. Currently the assignment is only reflected on the add and edit page.

If you want to go ahead and assign images to the tags, assign them using this prefix:
"icon/location_tag/" like "icon/location_tag/icon_name.png"

This will result in an image of http://...com/img/icon/location_tag/icon_name.png.

The rest of the project is using PNGs, so please use them so I don't have to convert. I'd shoot for 16x16 icons, but I'll leave the final decision for you. Also if you don't create the images, make sure they have a creative commons license that lets us use them on the site.

**Comment by Treijim** [comment]
Making a note, a new column of data needs to be added which specifies whether or not a person can place a story within this location.

**Comment by bheiskell** [comment]
Creating a new issue. I keep getting confused why this is open.
